survey-runner-queue/global_vars.tf

```diff
@@ -116,3 +116,7 @@ variable "logserver_cidr" {
 variable "audit_cidr" {
   description = "CIDR block of the centralised auditing service."
 }
+
+variable "delete_on_termination" {
```

Comment:
I think the variable should be something like `delete_volume_on_termination` as it's not clear outside of the context of the terraform file that this is for the volume.